src/index.tsx

```diff
@@ -81,63 +67,59 @@ const createReducer = (definition: Machine.Definition.Impl) => {
       }
 
       let selectedTransition =
-        typeof resolvedTransition === "string" ? { target: resolvedTransition } :
-        resolvedTransition.guard === undefined ? { target: resolvedTransition.target } :
-        resolvedTransition.guard({ context, event }) ? { target: resolvedTransition.target } :
-        { target: resolvedTransition.target, isDenied: true };
+        typeof resolvedTransition === "string"
+          ? { target: resolvedTransition }
+          : resolvedTransition.guard === undefined
+          ? { target: resolvedTransition.target }
+          : resolvedTransition.guard({ context, event })
+          ? { target: resolvedTransition.target }
+          : { target: resolvedTransition.target, isDenied: true };
```

Comment:
This is what I was talking about absolute dogshit 😂 Feel free to refactor into a non-ternary version because now it's no longer readable to me at least